IA-Policy.md

```diff
@@ -19,4 +19,14 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
 
 For information on roles and responsibilities, management commitment, coordination among organizational entities, compliance, reviews, and updates please see the [Technology Transformation Service's (TTS) Common Control Policy](https://github.com/18F/compliance-docs/blob/master/TTS-Common-Control-Policy.md).
 
-## Procedures
\ No newline at end of file
+## Procedures
+
+Identity and authentication in cloud.gov, and its underlying Amazon Web Service (AWS) account, is entirely gated by either cloud.gov's User Account and Authentication (UAA) Servers and their integration with GSA SecureAuth or by Amazon Web Service (AWS) Identity and Access Management (IAM).
```

Comment:
This feels like it should be two separate sentences, one for cloud.gov and one for AWS.